app/build.gradle

```diff
@@ -66,4 +70,5 @@ dependencies {
     implementation 'com.google.firebase:firebase-auth:20.0.1'
     implementation 'com.google.firebase:firebase-messaging:21.0.0'
     implementation 'com.journeyapps:zxing-android-embedded:4.1.0'
+    implementation 'com.android.volley:volley:1.1.1'
```

Comment:
what does this implementation do

Reply:
Allows for http requests through google's queue implementation

Reply:
oh i see

Reply:
https://developer.android.com/training/volley